app/models/manageiq/providers/openstack/inventory/parser/cloud_manager.rb

```diff
@@ -146,6 +147,43 @@ def host_aggregates
     end
   end
 
+  def placement_group_by_vm_id
+    @placement_group_by_vm_id ||= collector.placement_groups.each_with_object({}) { |sg, result| sg.members.each { |vm_id| result[vm_id] = sg } }
+  end
```

Comment:
This looks like a duplicate of https://github.com/ManageIQ/manageiq-providers-openstack/pull/806/files#diff-88d95edff4569231d0581d79243806e6ffe4badd621dc136e41e18e93ce615c9R68-R72

Can you just call `collector.server_group_by_vm_id` in the parser?

```suggestion
```